apps/etterlatte-brev-api/src/test/kotlin/no/nav/etterlatte/brev/BrevdistribuererTest.kt

```diff
@@ -52,7 +53,7 @@ class BrevdistribuererTest {
                 journalpostId,
                 DistribusjonsType.ANNET,
                 DistribusjonsTidspunktType.KJERNETID,
-                brev.mottaker.adresse,
+                brev.mottakere.first().adresse,
```

Comment:
Ser det er brukt `single()` andre steder. Regner med den fases ut når du legger til støtte for å "legge til". Men greit å være konsekvent.

Reply:
Enig! 👍